src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs

```diff
@@ -54,7 +54,7 @@ protected override void ConfigureHowToFindSaga(SagaPropertyMapper<RequestRespons
 
                 public class RequestResponseRequestingSagaData : ContainSagaData
                 {
-                    public virtual Guid CorrIdForResponse { get; set; } //wont be needed in the future
+                    public virtual Guid CorrIdForResponse { get; set; }
```

Comment:
I'm removing this test. This was a dead test for long and it has nothing to do with the persistence.